ComSemApp/templates/ComSemApp/student/assessment_results.html

```diff
@@ -54,17 +53,18 @@ <h1 class="card-title">Review complete!</h1>
                                     </td>
                                 </tr>
                                 <!--row that shows hidden information-->
-                                <tr id={{attempt.id}} class="hidden_row">  <!--row id corresponds to individual item id-->
+                                <tr id="row{{attempt.id}}" class="hidden_row">  <!--row id corresponds to individual item id-->
                                     <td colspan=4> <!-- merges all four columns in this row-->
                                         <p style="margin-left:40px"> <!--indentation for this paragraph-->
                                             <b>Correct Expression</b><br> 
                                             {{attempt.correct_formulation}} <br>
-                                            <b>Given Expression</b><br> 
+                                            <b>Problem Expression</b><br> 
```

Comment:
Not sure about this language change. We might want to bring this up with Dr. Hunter

Reply:
This is what dr.hunter requested!